packages/apps/stock/src/api/common/stock-move-line-api.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {axiosApiProvider, createStandardFetch} from '@axelor/aos-mobile-core';
+
+export async function addTrackingNumber({
+  stockMoveLineId,
+  version,
+  trackingNumber,
+}) {
+  return axiosApiProvider.post({
+    url: `/ws/rest/com.axelor.apps.stock.db.StockMoveLine/${stockMoveLineId}`,
+    data: {
+      data: {
+        id: stockMoveLineId,
+        version,
+        trackingNumber,
+      },
+    },
+  });
+}
+
+export async function fetchStockMoveLine({stockMoveLineId}) {
+  return createStandardFetch({
+    model: 'com.axelor.apps.stock.db.StockMoveLine',
+    id: stockMoveLineId,
+    fieldKey: 'stock_customerDeliveryLine',
+  });
+}
```

Comment:
I understand the idea but the fieldKey is not revelant. Moreover, this is not flexible enough if you need to add a related field only for one type of stock move line. Please replace this with functions in the relevant stock move line api file.